plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/BaseBigQueryConnectorTest.java

```diff
@@ -59,12 +60,17 @@ public abstract class BaseBigQueryConnectorTest
     protected BigQuerySqlExecutor bigQuerySqlExecutor;
     private String gcpStorageBucket;
 
+    private String bigQueryConnectionId;
+
     @BeforeAll
     public void initBigQueryExecutor()
     {
         this.bigQuerySqlExecutor = new BigQuerySqlExecutor();
         // Prerequisite: upload region.csv in resources directory to gs://{testing.gcp-storage-bucket}/tpch/tiny/region.csv
         this.gcpStorageBucket = System.getProperty("testing.gcp-storage-bucket");
+        this.bigQueryConnectionId = requireNonNull(
+                System.getProperty("testing.bigquery-connection-id"),
+                "testing.bigquery-connection-id is not set");
```

Comment:
Usually we have such code in trino in one line. 
Also, how is it populated on CI?
I see you've updated documentation with `testing.bigquery-connection-id=my_project.us.connection-id`, but there are no corresponding changes within `./.github/workflows/ci.yml` as they are for
`bigquery.credentials-key`, `testing.gcp-storage-bucket`, `testing.alternate-bq-project-id`